src/content/overview/introduction.md

```diff
@@ -0,0 +1,34 @@
+  - How does Giant Swarm help engineering teams?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+last_review_date: 2024-03-12
+---
+
+Giant Swarm's mission is to empower our customers to build their own cloud-native [developer platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#what-is-a-platform). As engineering teams grow and their environments expand, they often face exponentially growing complexity within the cloud-native ecosystem. This can be overwhelming. We believe that in most cases, having a [unified platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) plays a crucial role in keeping or gaining control over the entropic nature of such complex environments, while also significantly enhancing the users experience in using them.
+
+Ultimately, developers want to deploy an app into production without having to worry too much about security or compliance, but still have good monitoring and deployment options. Our goal is to help the engineers who build and maintain the platform meet developers' expectations without compromising their organizations' security posture.
+
+Giant Swarm addresses this challenge by providing a flexible base platform. The platform enables technology departments to ensure governance and control over development processes and operations, while allowing developers to smoothly manage their development cycle. Check out the documentation below to learn how we make this possible.
+
+## Our platform your platform
+
+You may be wondering: How does Giant Swarm achieve flexibility and standardization at the same time? Giant Swarm does not offer a one-size-fits-all platform that, in the most optimistic scenario, covers 80% of use cases and ignores the rest of your business. Instead, we trust the cloud-native ecosystem to provide a set of features that help platform teams build a developer platform that fits the company's constraints, processes, and unique features. We believe that there is no universal [internal developer platform](https://internaldeveloperplatform.org/) that can cover all needs, especially in the enterprise world, and for this reason we make our solution composable and customizable.
+
+Our approach gives you the ability to define team-specific policies, flexible application delivery processes, or different traffic routing setups. We help you create templates tailored to your team's needs while ensuring excellent visibility and control over their activities.
+
+## Working with our customers
+
+We offer a product that lets you start quickly with a production-ready platform. But our team works with your engineers every day to build something together from this starting point. There is a common baseline that your security and governance team can trust upon. And specific requirements will be discussed and configured in collaboration. Good communication and support is key and this is exactly what we are good at.
```

Comment:
Maybe add something like us having long-standing customers who trust us, linking to use cases on the `giantswarm.io` website?